massa-client/src/settings.rs

```diff
@@ -10,6 +10,7 @@ lazy_static::lazy_static! {
     pub static ref SETTINGS: Settings = build_massa_settings("massa-client", "MASSA_CLIENT");
 }
 
+#[allow(dead_code)]
```

Comment:
There are several new dead codes. Do you know why this is needed? Is it behind a feature that is not detected?

Reply:
I suggest merging this PR and add a meta issue tracking all of these clippy allow directives and remove/comment them. What do you think?

Reply:
Seems good to me!

Reply:
https://github.com/massalabs/massa/issues/4752